java/src/jmri/jmrit/display/Positionable.java

```diff
@@ -104,18 +104,60 @@
      */
     public String getNameString();
 
+    /**
+     * Add additional menu items to the menu.
+     *
+     * @param popup the menu to add
+     * @return true if adding items; false otherwise
```

Comment:
I think the `popup` parameter is the `the menu to which the items will be added`, which is slightly different.  (multiple places)